packages/create-cloudflare/README.md

```diff
@@ -2,58 +2,7 @@
 
 A CLI for creating and deploying new applications to [Cloudflare](https://developers.cloudflare.com/).
 
-## Usage
```

Comment:
@admah please let me know if this change is ok with you 🙏 

As I [mentioned in the docs PR](https://github.com/cloudflare/cloudflare-docs/pull/11394#issuecomment-1776793445) I think that it would be a good strategy to avoid duplicating informations and have everything in the Cloudflare docs and just point to that.

This does make it less immediate on npm (forcing people to click and navigate to the docs page) so if you prefer to duplicate the information I understand 🙂 (but this to me feels like a much more maintainable solution)

Reply:
This is great. I think this is a good solution that points users to more info, while keeping it maintainable in the long run.